composer.json

```diff
@@ -35,5 +42,8 @@
       "src/WordPress/Zip/functions.php",
       "src/WordPress/Streams/stream_str_replace.php"
     ]
+  },
+  "scripts": {
+    "phpcs": "phpcs --standard=PSR4"
```

Comment:
What would it take to use [WordPress coding standards](https://github.com/WordPress/WordPress-Coding-Standards) here?

Reply:
Done. But linting and final standards adjustments will be done in a subsequent PR.